modules/qna/src/views/full/index.tsx

```diff
@@ -44,9 +44,22 @@ export default class QnaAdmin extends Component<Props> {
 
   fetchFlows() {
     this.props.bp.axios.get('/flows').then(({ data }) => {
-      const flowsList = data
-        .filter(flow => !flow.name.startsWith('skills/'))
-        .map(({ name }) => ({ label: name, value: name }))
+      const reorderFlows = flows =>
+        [
+          flows.find(x => x.name === 'main'),
+          flows.find(x => x.name === 'error'),
+          flows.find(x => x.name === 'timeout'),
+          ...flows.filter(x => !['main', 'error', 'timeout'].includes(x.name))
+        ].filter(x => Boolean(x))
+
+      const getFlowLabel = (name: string) =>
+        name === 'main' ? 'Main' : name == 'error' ? 'Error handling' : name == 'timeout' ? 'Timeout' : name
+
+      const flowsList = reorderFlows(
+        data
+          .filter(flow => !flow.name.startsWith('skills/'))
+          .map(({ name }) => ({ name: name.replace(/.flow.json$/i, '') }))
+      ).map(({ name }) => ({ label: getFlowLabel(name), value: name }))
```

Comment:
There are already a bunch of stuff happening here, I suggest simplifying it a bit:
```js
const flowsList = data....
const orderedList = reorderFlows(flowsList ).map(x...)
```

Reply:
You are right, it is over-complicated